py/mkenv.mk

```diff
@@ -29,6 +29,11 @@ else
 Q =
 endif
 
+# depend_var is a function that can be used to add a virtual dependency
+# to a target based on the given make variable.
+# eg. $(OBJ): $(call depend_var, MY_VAR)
+depend_var = $(BUILD)/dependvar/$(strip $(1))\#$(shell echo $($(strip $(1))) | md5sum | cut -d ' ' -f 1)
```

Comment:
Is the comment `#` escaped on this line?

Reply:
That hash was used on purpose as the marker character between "argname" and "arg value hash" in the generated filename. In hindsight it's a poor choice for separator character because it took me a while to remember why I'd used a hash here...